ui/src/map/map-view.tsx

```diff
@@ -228,10 +228,11 @@ const MapView: React.FC<MapViewProps> = ({
         const clusterPoint = selection.selectedItems.clusterPoints[0];
 
         if (!olMap || !clusterPoint) return;
-        const pos = [clusterPoint.x + 0.5, clusterPoint.y];
+        const pos = [clusterPoint.x, clusterPoint.y];
         const popupElement = document.getElementById('clusteroverlay') || undefined;
         const popup = new Overlay({
             position: pos,
+            offset: [7, 0],
```

Comment:
Openlayers tarjoaa `offset`-propertyn tällaisiin käyttötilanteisiin. `Position` on suoraan karttayksiköitä, ja tämä taas on pikseleitä -> menu ei liiku zoom-tason mukana, vaan on aina tällä offsetillä pylpyrän keskipisteestä. 7 pikseliä valikoitui offset-arvoksi sillä perusteella, että se on myös cluster pointin ympyrän säde -> nyt menu juuri ja juuri koskee cluster pointia, mutta ei kuitenkaan oikeastaan ole sen päällä